chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts

```diff
@@ -0,0 +1,155 @@
+import {Pick3DrawDateCardDomain} from "./pick3-draw-date-card.domain";
+import {TestBed} from "@angular/core/testing";
+import {NumberUtilityService} from "../services/numberUtility.service";
+import {Pick3DrawTimeEnum} from "./pick3-draw-time.enum";
+import {Pick3LotteryService} from "../services/pick3-lottery.service";
+import {IonicToastNotificationService} from "../services/ionic-toast-notification.service";
+describe('Pick3DrawDateCardDomain', () => {
+
+  beforeEach(() => {
+
+  });
+  let model = new Pick3DrawDateCardDomain({
+    drawDate : new Date(),
+    drawState : 'gotoHome',
+    drawTime: Pick3DrawTimeEnum.Pick3DrawTimeEnum.MORNING,
+    drawTimeAsString: "Morning",
+    upcomingDrawTime: new Date(),
+    hasWinner: false,
+    backgroundImage: Pick3LotteryService,
+    winningNumber: 462,
+    winningNumberDigits: [4,6,2],
+    drawDateIcon: new Date(),
+    slideNumber: 7
+
+  });
+
+  // drawDate
+  it('should have a drawDate defined', function () {
+    const drawDate = model.getDrawDate();
+    expect(drawDate).toBeDefined();
```

Comment:
In addition to this, you an expect for exact date previous created.  See comments above.

Reply:
Renamed and inserted use for date